### PR TITLE
Changes couchdb service names to couch-x.local

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

medic/cht-core#7962

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:7962-valid-couchdb-service-name/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:7962-valid-couchdb-service-name/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:7962-valid-couchdb-service-name/docker-compose/cht-couchdb-clustered.yml)
 
# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: scripts/build/cht-couchdb-cluster.yml.template [hunk 1]
```diff
@@ -11,8 +11,8 @@ services:
       - "COUCHDB_PASSWORD=${COUCHDB_PASSWORD:?COUCHDB_PASSWORD must be set}"
       - "COUCHDB_SECRET=${COUCHDB_SECRET}"
       - "COUCHDB_UUID=${COUCHDB_UUID}"
-      - "SVC_NAME=${SVC1_NAME:-couchdb.1}"
-      - "CLUSTER_PEER_IPS=couchdb.2,couchdb.3"
+      - "SVC_NAME=${SVC1_NAME:-couchdb-1.local}"
+      - "CLUSTER_PEER_IPS=couchdb-2.local,couchdb-3.local"
```
Comment (garethbowen) [inline scripts/build/cht-couchdb-cluster.yml.template:15]: Shouldn't this use the env var if given, falling back to the default?

Comment (dianabarsan) [reply scripts/build/cht-couchdb-cluster.yml.template:15]: oh, damn, yes it should!

**Comment by dianabarsan** [comment]
Oh, I just realized I've been pushing commits to the wrong branch :( 
I'll revert them.

**Review by garethbowen — APPROVED** [review]
LGTM! Nice one.

**Comment by mrjones-plip** [comment]
@dianabarsan - I expected no instances of `couchdb.1` to be left, but I found 5 files with this string.  No need to block this PR because of it, but just wanted to call it out in case it's important:

```
➜  cht-core git:(7962-valid-couchdb-service-name) grep -r 'couchdb\.1' .             
./couchdb/prod.couchdb-cluster.yml:  couchdb.1:
./couchdb/prod.couchdb-cluster.yml:      - "SVC_NAME=${SVC1_NAME:-couchdb.1}"
./couchdb/prod.couchdb-cluster.yml:      - "COUCHDB_SYNC_ADMINS_NODE=${COUCHDB_SYNC_ADMINS_NODE:-couchdb.1}"
./couchdb/prod.couchdb-cluster.yml:      - "COUCHDB_SYNC_ADMINS_NODE=${COUCHDB_SYNC_ADMINS_NODE:-couchdb.1}"
./couchdb/prod.couchdb-cluster.yml:      - "COUCHDB1_SERVER=${COUCHDB1_SERVER:-couchdb.1}"
./couchdb/README.md:This field ensures that the hash that CouchDB generates during bootup for saving your admin password syncs across nodes in the cluster. This sync of the hashed password is required for user session cookies to work across nodes. We omit this field from our setup-coordination-node, which is commonly couchdb.1. This is a required field for the other nodes in your cluster.
./couchdb/test.couchdb-cluster.yml:  couchdb.1:
./couchdb/test.couchdb-cluster.yml:      - "SVC_NAME=couchdb.1"
./couchdb/test.couchdb-cluster.yml:      - "COUCHDB_SYNC_ADMINS_NODE=${COUCHDB_SYNC_ADMINS_NODE:-couchdb.1}"
./couchdb/test.couchdb-cluster.yml:      - "COUCHDB_SYNC_ADMINS_NODE=${COUCHDB_SYNC_ADMINS_NODE:-couchdb.1}"
./couchdb/test.couchdb-cluster.yml:    - couchdb.1
./couchdb/test.couchdb-cluster.yml:      - "SVC_NAME=couchdb.1"
./couchdb/tests/tests.bats:    assert_output --partial '{"all_nodes":["couchdb@couchdb.1","couchdb@couchdb.2","couchdb@couchdb.3"],"cluster_nodes":["couchdb@couchdb.1","couchdb@couchdb.2","couchdb@couchdb.3"]}'
./couchdb/tests/tests.bats:    run curl -s GET http://$COUCHDB_USER:$COUCHDB_PASSWORD@couchdb.1:5984/_up
./couchdb/tests/tests.bats:    run curl -s -X PUT http://$COUCHDB_USER:$COUCHDB_PASSWORD@couchdb.1:5984/tests
./haproxy-healthcheck/check.py:# Example: COUCHDB_SERVERS="couchdb.1,couchdb.2,couchdb.3"
```

**Comment by dianabarsan** [comment]
I think those are just part of the CouchDb tests. afaict nothing uses those files at the moment.

**Pull request merged**